tests/tests/test_train_schedule.py

```diff
@@ -765,7 +765,7 @@ def test_etcs_schedule_result_slowdowns(
     _assert_equal_speeds(speed_after_brake_142_to_120, SPEED_LIMIT_112)
 
     # Slowdown for Safety Speed stop: should probably disappear for ETCS at some point.
-    offset_start_brake_120_to_30 = 45_635_550
+    offset_start_brake_120_to_30 = 45_635_533
```

Comment:
Initially wrongly named:
```suggestion
    offset_start_brake_112_to_30 = 45_635_533
```

Reply:
Fixed in https://github.com/OpenRailAssociation/osrd/pull/11616/commits/a0a6e86f8a5dbe21c03d2a4c9afecf9fe8ae08b9